packages/renderer/src/lib/ui/LinearProgress.svelte

```diff
@@ -0,0 +1,64 @@
+<style>
```

Comment:
I'm not sure of the styles and why it's not using tailwind

I mean, all of these
`
    appearance: none;
    border: none;
    height: 0.10em;
    color: var(--pf-global--primary-color--100);
    font-size: 16px;
`
can use tailwind properties